src/sentry/static/sentry/app/components/placeholder.tsx

```diff
@@ -4,33 +4,45 @@ import PropTypes from 'prop-types';
 
 import space from 'app/styles/space';
 
-const Placeholder = styled(props => {
+const defaultProps = {
+  shape: 'rect' as 'rect' | 'circle',
+  bottomGutter: 0,
+  width: '100%',
+  height: '60px',
+};
+
+type DefaultProps = Readonly<typeof defaultProps>;
+
+type Props = {
+  className?: string;
+} & Partial<DefaultProps>;
+
+const Placeholder = styled((props: Props) => {
   const {className} = props;
   return <div data-test-id="loading-placeholder" className={className} />;
-})`
+})<Props>`
   display: flex;
   flex-direction: column;
   flex-shrink: 0;
   justify-content: center;
 
   background-color: ${p => p.theme.placeholderBackground};
-  width: ${p => p.width};
-  height: ${p => p.height};
+  width: ${p => p.width || '100%'};
```

Comment:
Isn't this default?

Reply:
I'll remove this. I put this in before I figured out how to get defaultProps to typecheck.